package.json

```diff
@@ -7,18 +7,18 @@
     "type": "git",
     "url": "https://github.com/DEFRA/ffc-demo-web.git"
   },
-  "main": "index.js",
+  "main": "app/index.js",
   "scripts": {
     "build": "npm run build:css",
-    "build:css": "node-sass --output-style=expanded --output=server/public/build/stylesheets client/sass",
-    "build:css-watch": "node-sass --output-style=expanded --output=server/public/build/stylesheets --watch client/sass",
+    "build:css": "node-sass --output-style=expanded --output=app/static/stylesheets app/static/stylesheets",
+    "build:css-watch": "node-sass --output-style=expanded --output=app/static/stylesheets --source-map true --watch app/static/stylesheets",
```

Comment:
The original approach to Sass compiling seemed unintuitive as it put the CSS into a deeply nested folder within `server/` that doesn't reflect the path to the original Sass location.

Arguably, we should compile or copy the entire application from the source folder to a sibling build (or dist) folder. That would include copying the JavaScript and robots.txt directly from source to build unchanged, which seems silly but would mean both source and build folders contain a complete set of application code.

Since this is only a demonstration of the proposed repository structure, I've taken the easy option of simply compiling the Sass in-place for now so that all the application code sits in the same top-level folder. I think it ends up looking pretty clear so may be fine as it is. If we do introduce a build/dist folder, that would only ever exist within the container, not in the source repository.

Reply:
nice changes, long term a dist folder may be a good approach, it's good to separate transpiled source out, even if it is a bit of a mix.